### DISCUSSION TITLE
建议在关闭WAF开关后停止记录请求日志

### DISCUSSION THREAD

**Discussion by sch-lda** [1Panel-dev/1Panel#6532] [Q&A]
### 1Panel 版本

1.10.15-lts

### 请描述您的需求或者改进建议

1~2个月前，由于在其他设备使用了基于反代的防火墙，我关闭了1Panel WAF的总开关，但/opt/1panel/apps/openresty/openresty/1pwaf/data/db/sites/[域名]/ 目录下的site_req_logs.db大小仍在快速增加，由于请求地址中包含大量字符串，月初删除以后又积累了1.4GB. 用sqlite browser查看db文件能够看出记录的ip、地理位置、UA等信息，localtime也显示确实是新记录的数据
![image](https://github.com/user-attachments/assets/6c3f86a9-f4cf-46c0-8f24-a55a4c30748d)
![image](https://github.com/user-attachments/assets/1d997e24-0c7c-4f78-8976-c22c45c7d342)

建议在用户关闭WAF后停止记录WAF日志，或者添加滚动记录/大小限制选项

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们先在本地环境验证一下上述问题。

**Comment by zhengkunwang223** [comment]
site_req_logs.db 是网站监控生成的数据 可能是以前的版本错误的开启了网站监控 
你可以修改 
![image](https://github.com/user-attachments/assets/7e9564f1-2117-4cd0-b399-1152916662c9)
此处 把 on 改为 off
